# --- a/netsurf/content/handlers/javascript/duktape/Makefile
# +++ b/netsurf/content/handlers/javascript/duktape/Makefile
--- a/content/handlers/javascript/duktape/Makefile
+++ b/content/handlers/javascript/duktape/Makefile
@@ -15,14 +15,14 @@
 $(OBJROOT)/duktape/generics.js.inc: content/handlers/javascript/duktape/generics.js
 	$(Q)$(MKDIR) -p $(OBJROOT)/duktape
 	$(VQ)echo "     XXD: $<"
-	$(Q)xxd -i $< $@.tmp
+	$(Q)xxd -i $< > $@.tmp
 	$(Q)sed -e 's/content_handlers_javascript_duktape_generics_js/generics_js/' $@.tmp > $@
 
 # Generator for the C include representing the polyfill.js
 $(OBJROOT)/duktape/polyfill.js.inc: content/handlers/javascript/duktape/polyfill.js
 	$(Q)$(MKDIR) -p $(OBJROOT)/duktape
 	$(VQ)echo "     XXD: $<"
-	$(Q)xxd -i $< $@.tmp
+	$(Q)xxd -i $< > $@.tmp
 	$(Q)sed -e 's/content_handlers_javascript_duktape_polyfill_js/polyfill_js/' $@.tmp > $@
 
 # ensure genbind generates debugging files
